### DISCUSSION TITLE
Cookbook idea

### DISCUSSION THREAD

**Discussion by AndreaCrotti** [babashka/babashka#856] [Ideas]
It would be nice if as part of the documentation, there was a simple guide/cookbook to show examples of conversions from bash scripts to babashka scripts.

The examples don't need to be complicated, and maybe they can also just be taken from somewhere like
https://linuxhint.com/30_bash_script_examples/
and it would be a good way to introduce anyone who knows a bit of bash but nothing about Clojure to babashka.

**Comment by borkdude** [comment]
Work started here: https://github.com/babashka/babashka/wiki/Tasks:-Bash-and-Babashka-equivalents

Feel free to contribute.
